### PR TITLE
Integrated internal changes from Google

### PR DESCRIPTION
**Pull request by acozzette**

### CODE REVIEW

**Comment by acozzette** [comment]
retest this please

**Comment by acozzette** [comment]
retest this please

File: js/commonjs/export_testdeps.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ goog.require('jspb.arith.UInt64');
 goog.require('jspb.BinaryEncoder');
 goog.require('jspb.BinaryDecoder');
 goog.require('jspb.BinaryWriter');
+goog.require('goog.testing.PropertyReplacer');
```
Comment (xfxyjwf) [inline js/commonjs/export_testdeps.js]: Sort this?

File: js/message_test.js [hunk 2]
```diff
@@ -1052,13 +1052,4 @@ describe('Message test suite', function() {
     assertNan(message.getDefaultDoubleField());
   });
 
-  it('testSerializePreservesEmptyNestedField', function() {
```
Comment (xfxyjwf) [inline js/message_test.js]: Why is this test removed?

File: js/commonjs/export_testdeps.js (hunk 1 above)
Comment (acozzette) [reply js/commonjs/export_testdeps.js]: Done

File: js/message_test.js (hunk 2 above)
Comment (acozzette) [reply js/message_test.js]: This one tries to use the internal-only JSPB wire format.

**Comment by acozzette** [comment]
Thanks, @xfxyjwf! I'll merge this now without waiting for Travis, since this includes only a trivial change since the previous successful Travis run.

**Pull request merged**